### PR TITLE
Section/zkcloud snarkvm

### PR DESCRIPTION
**Pull request by Zack-Xb**
- zkCloud overview
- snarkOs overview, setup guide and developer guide.
- snarkvm started overview 

**TODO:**
- snarkos developer practical examples that they can copy and run
- snarkvm more technical side 
- overview of Varuna + link to paper

Opening this PR so the review of what has been done till now can start and I will continue committing with the TODOs and iterations on the feedback.

### CODE REVIEW

File: src/learn/zkcloud/README.md [hunk 1]
```diff
@@ -1 +1,6 @@
 # zkCloud
+zkCloud is the infrastructure that allows Aleo to enable decentralized private computations. This infrastructure is made up of two parts, `snarkVm` - zero knowledge off chain execution client and `snarkOs` which is the network client that verifies that off chain computation has been done correctly by verifying the `zksnark` that has been submitted.
```
Comment (zklim) [inline src/learn/zkcloud/README.md:2]: Standardise formatting of terms like `snarkOS` and `snarkVM`

File: src/learn/zkcloud/snarkos_dev_guide.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+* [3. Local Devnet](#3-local-devnet)
+  * [3.1 Install `tmux`](#31-install-tmux)
+  * [3.2 Start a Local Devnet](#31-start-a-local-devnet)
+  * [3.3 View a Local Devnet](#31-view-a-local-devnet)
+  * [3.4 Stop a Local Devnet](#31-stop-a-local-devnet)
+
+
+### 1. Quick Start
+
+In the first terminal, start the first validator by running:
```
Comment (zklim) [inline src/learn/zkcloud/snarkos_dev_guide.md:21]: Maybe add reason why we need to start 4 validators at minimum.

File: src/learn/zkcloud/snarkos_dev_guide.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+#### 3.2 Start a Local Devnet
+
+To start a local devnet, run:
+```bash
+./devnet.sh
+```
+Follow the instructions in the terminal to start the devnet.
+
+#### 3.3 View a Local Devnet
```
Comment (zklim) [inline src/learn/zkcloud/snarkos_dev_guide.md:178]: Suggestion: Add more frequently used tmux command for easier navigation. Like `ctrl+b [` to scroll and `q` to quit scrolling as example.

File: src/learn/zkcloud/README.md [hunk 4]
```diff
@@ -1 +1,6 @@
 # zkCloud
+zkCloud is the infrastructure that allows Aleo to enable decentralized private computations. This infrastructure is made up of two parts, `snarkVM` - zero knowledge off chain execution client and `snarkOS` which is the network client that verifies that off chain computation has been done correctly by verifying the `zksnark` that has been submitted.
+
+Together `snarkVM` and `snarkOS` allow individuals (shielded identities) to remain private and use this off chain computation (shielded transactions) to interact with a given decentralized application (program) on Aleo. On the other hand it also enables developers to create these privacy preserving decentralized applications through using leo or aleo instructions and deploying the program to the network.
```
Comment (ramidecodes) [inline src/learn/zkcloud/README.md:4]: The parenthesis explaining shielded identities and shielded transactions are a bit misleading. 
A shielded transactions is one where the owner and amount are encrypted, not an off-chain computation as written here. A transaction can be shielded (aka private) due to the Record Model and other architectural choices of Aleo, executing computation off-chain is only part of what enables a shielded transaction.

Comment (ramidecodes) [reply src/learn/zkcloud/README.md:4]: I like how this article explains the intuition of Aleo's zkCloud without going deep into the technical explanation. We could use some of this explanations:
https://aleo.org/post/zkcloud/

File: src/learn/zkcloud/snarkos.md [hunk 5]
```diff
@@ -1 +1,26 @@
-# snarkOS
+# SnarkOS
+
+>SnarkOS is a decentralized operating system for zero-knowledge applications. This code forms the backbone of Aleo network, which verifies transactions and stores the encrypted state applications in a publicly-verifiable manner.
```
Comment (ramidecodes) [inline src/learn/zkcloud/snarkos.md:3]: Is it missing a word? "...stores the encrypted state OF applications in a publicly-verifiable manner...".

File: src/learn/zkcloud/snarkos_build_guide.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+  * [1.2 Installation](#12-installation)
+* [2. Run an Aleo Node](#2-run-an-aleo-node)
+  * [2.1 Run an Aleo Client](#21-run-an-aleo-client)
+  * [2.2 Run an Aleo Prover](#22-run-an-aleo-prover)
+* [3. Command Line Interface](#3-command-line-interface)
+* [4. FAQs](#4-faqs)
+## 1. Build Guide
+### 1.1 Requirements
+
+The following are **minimum** requirements to run an Aleo node:
```
Comment (ramidecodes) [inline src/learn/zkcloud/snarkos_build_guide.md:16]: How are we going to track changes in the original documentation when this requirements change?

**Review by ramidecodes — APPROVED** [review]
Left some comments suggesting typo fixes and general improvements.

File: src/learn/zkcloud/snarkos_build_guide.md (hunk 6 above)
Comment (Zack-Xb) [reply src/learn/zkcloud/snarkos_build_guide.md:16]: I think best have someone clearly designate to update the requirements if and when they change.

**Review by ramidecodes — APPROVED** [review]

**Pull request merged**